ospf6d/ospf6_zebra.c

```diff
@@ -371,6 +372,38 @@ DEFUN(show_zebra,
 	return CMD_SUCCESS;
 }
 
+static void ospf6_zebra_append_opaque_attr(struct ospf6_route *request,
+					   struct zapi_route *api)
+{
+	struct ospf_zebra_opaque ospf_opaque = {};
+
+	/* OSPF path type */
+	snprintf(ospf_opaque.path_type, sizeof(ospf_opaque.path_type), "%s",
+		 OSPF6_PATH_TYPE_NAME(request->path.type));
```

Comment:
same as a `strlcpy`, no? `strlcpy` ought to be marginally faster (not that it matters here)